### ISSUE TITLE
'min_size' Style property not working

### ISSUE THREAD

**Issue by vav-dev** [bevyengine/bevy#8169]
## Bevy version

bevy 0.10.0

## What you did

I upgrade my bevy project from 0.9 to 0.10

## What went wrong

The 'min_size' Style property of the NodeBundle does not seem to be taken into account:

```rust
NodeBundle {
    style: Style {
        min_size: Size {
            height: Val::Px(100.),
            width: Val::Px(250.),
        },
        ..default()
    },
    ..default()
}
```

Give this result:

![image](https://user-images.githubusercontent.com/33455670/227060858-cd5eae99-f093-466b-b174-a1d3098ed8d2.png)

Instead of this:

![image](https://user-images.githubusercontent.com/33455670/227061142-d5d272df-5a64-4f7b-9341-47c37e4ac850.png)

This can be fixed replacing 'min_size' by 'size' but it loses all the 'responsivness'

**Comment by mockersf** [comment]
this has probably been fixed by https://github.com/bevyengine/bevy/pull/7948

**Comment by nicoburns** [comment]
This should be fixed in bevy `0.10.1` :)
